### PR TITLE
Readded serialisation of GraphML updated to work with new internals.

### PR DESCRIPTION
**Pull request by thisiswhereitype**
Haddock and examples included.

### CODE REVIEW

**Comment by thisiswhereitype** [comment]
Pending working out why all the commits were included.

File: src/Pangraph.hs [hunk 1]
```diff
@@ -28,17 +28,17 @@ import Data.Maybe            (mapMaybe, isJust)
...
 import qualified Data.Map.Strict  as Map
 import qualified Data.ByteString  as BS
-{-| The 'Pangraph' type is abstract and intended to be the core intermediate type between abstract representations of other graph libaries and the results of parsers. -}
+{-| The 'Pangraph' type is abstract and the core intermediate type between abstract representations of graphs. -}
 data Pangraph = Pangraph
   { vertices' :: Map VertexID Vertex
   , edges' :: Map EdgeID Edge
   } deriving (Eq)
-{-| A 'Vertex' holds `Attributes` and must have a unique `VertexID` to be constructed -}
+{-| A Vertex holds `Attributes` and must have a unique `VertexID` to be constructed with `makeVertex`. -}
```
Comment (snowleopard) [inline src/Pangraph.hs:36]: I don't think Haddock supports backticks? Use single quotes to link to other definitions, e.g. `holds a list of 'Attribute' values`, `'VertexID'` and `'makeVertex'`.

**Comment by snowleopard** [comment]
@thisiswhereitype I've still added a comment -- I don't think backticks are supported by Haddock (or am I wrong?), so use single quotes to link to other definitions.

**Comment by thisiswhereitype** [comment]
You were correct :) Reopened in #16 due to forced commit to fix this.

**Pull request closed without merging**